src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/Handlers/SourceItemHandler.cs

```diff
@@ -121,6 +121,8 @@ public override Task OnContextReleasedAsync(IWorkspaceProjectContext context)
 
         private void RemoveSourceFile(string fullPath, IWorkspaceProjectContext context)
         {
+            fullPath = _project.MakeRooted(fullPath);
```

Comment:
@srivatsn I figured out I cannot still add unit tests for `ILanguageServiceCommandLineHandler` as the Handle method takes `IWorkspaceProjectContext` which doesn't yet give IVT access to our unit test projects.